lighthouse-core/test/lib/timing-trace-saver-test.js

```diff
@@ -37,92 +36,11 @@ const mockEntries = [{
   entryType: 'measure',
 },
 ];
-const expectedTrace = {
-  traceEvents: [{
-    name: 'lh:init:config',
-    cat: 'measure',
-    ts: 650000,
-    dur: 210000,
-    args: {},
-    pid: 0,
-    tid: 50,
-    ph: 'X',
-    id: '0x0',
-  },
-  {
-    name: 'lh:runner:run',
-    cat: 'measure',
-    ts: 870000,
-    dur: 120000,
-    args: {},
-    pid: 0,
-    tid: 50,
-    ph: 'X',
-    id: '0x1',
-  },
-  {
-    name: 'lh:runner:auditing',
-    cat: 'measure',
-    ts: 990000,
-    dur: 750000,
-    args: {},
-    pid: 0,
-    tid: 50,
-    ph: 'X',
-    id: '0x2',
-  },
-  {
-    name: 'lh:audit:is-on-https',
-    cat: 'measure',
-    ts: 1010000,
-    dur: 10000,
-    args: {},
-    pid: 0,
-    tid: 50,
-    ph: 'X',
-    id: '0x3',
-  },
-  ],
-};
-
 
 describe('generateTraceEvents', () => {
-  let consoleError;
-  let origConsoleError;
-
-  beforeEach(() => {
-    origConsoleError = console.error;
-    consoleError = jest.fn();
-    console.error = consoleError;
-  });
-
-  afterEach(() => {
-    console.error = origConsoleError;
-  });
-
-  it('generates a single trace event', () => {
-    const event = generateTraceEvents(mockEntries);
-    assert.deepStrictEqual(event.slice(0, 1), expectedTrace.traceEvents.slice(0, 1));
-  });
-
-  it('doesn\'t allow overlapping events', () => {
-    const overlappingEntries = [{
-      startTime: 10,
-      name: 'overlap1',
-      duration: 100,
-      entryType: 'measure',
-    },
-    {
-      startTime: 30,
-      name: 'overlap2',
-      duration: 100,
-      entryType: 'measure',
-    },
-    ];
-
-    generateTraceEvents(overlappingEntries);
-    expect(consoleError).toHaveBeenCalled();
-    expect(consoleError.mock.calls[0][0]).toContain('measures overlap');
+  it('generates a pair of trace events', () => {
+    const events = generateTraceEvents([mockEntries[0]]);
+    expect(events.slice(0, 1)).toMatchSnapshot();
```

Comment:
seems a bit odd that we call the test `generates a pair` but then only look at 1 event, just do 'em both?

Reply:
lol thx